slf4j-api/src/main/java/org/slf4j/helpers/BasicMDCAdapter.java

```diff
@@ -86,10 +95,11 @@ public void put(String key, String val) {
   /**
    * Get the context identified by the <code>key</code> parameter.
    */
+  @Override
   public String get(String key) {
-    Map<String, String> Map = (Map<String, String>) inheritableThreadLocal.get();
+    Map<String, String> Map = inheritableThreadLocal.get();
```

Comment:
I know this was pre-existing, but the variable is capitalized Map, but should be just map.